content/shared/js/skipto.js

```diff
@@ -483,177 +1339,830 @@ $skipToId [role="menuitem"].hover .label {
+      const borderOffset = -1 * rect.height + 3 + 'px';
+      containerNode.style.setProperty('--skipto-show-border-offset', borderOffset);
+    }
+    else {
+      if (menuButtonNode.classList.contains('popup')) {
+        const popupOffset = -1 * rect.height + 'px';
+        containerNode.style.setProperty('--skipto-popup-offset', popupOffset);
+      }
+    }
+    containerNode.style.setProperty('--skipto-menu-offset', rect.height + 'px');
+
+    const zIndex2 = config.zIndex ?
+                    (parseInt(config.zIndex) + 1).toString() :
+                    '2000001';
+
+    updateStyle(containerNode, '--skipto-z-index-2', zIndex2, '');
+
+    const zIndexHighlight = config.zIndex ?
+                    (parseInt(config.zIndex) - 1).toString() :
+                    '199999';
+
+    updateStyle(containerNode, '--skipto-z-index-highlight', zIndexHighlight, '');
+
+    // Special case for theme configuration used in Illinois theme
+    if (typeof theme.highlightTarget === 'string') {
+      config.highlightTarget = theme.highlightTarget;
+    }
 
   }
 
   /*
-   *   @function enderStyleElement
+   *   @function renderStyleElement
    *
    *   @desc  Updates the style sheet template and then attaches it to the document
    *
-   * @param  {Object}  colorThemes     -  Object with theme information
+   * @param  {Object}  attachNode      - DOM element node to attach button and menu container element
    * @param  {Object}  config          -  Configuration information object
-   * @param  {String}  skipYToStyleId  -  Id used for the skipto container element
+   * @param  {Boolean} useURLTheme     - When true use the theme associated with the URL
    */
-  function renderStyleElement (colorThemes, config, skipToId) {
-    styleTemplate.innerHTML = styleTemplate.innerHTML.replaceAll('$skipToId', '#' + skipToId);
-    addCSSColors(colorThemes, config);
-    const styleNode = styleTemplate.content.cloneNode(true);
-    styleNode.id = `${skipToId}-style`;
-    const headNode = document.getElementsByTagName('head')[0];
-    headNode.appendChild(styleNode);
+  function renderStyleElement (attachNode, config, useURLTheme=false) {
+    let styleNode = attachNode.querySelector(`style`);
+    if (!styleNode) {
+      styleNode = document.createElement('style');
+      attachNode.appendChild(styleNode);
+      styleNode.textContent = cssStyleTemplate.textContent;
+    }
+
+    const containerNode = attachNode.querySelector('.container');
+
+    updateCSS(containerNode, config, useURLTheme);
+
   }
 
-  /* utils.js */
+  /* shortcutInfoDialog.js */
 
   /* Constants */
-  const debug$5 = new DebugLogging('Utils', false);
-  debug$5.flag = false;
+  const debug$a = new DebugLogging('[shortcutsInfoDialog]', false);
+  debug$a.flag = false;
 
+  const templateInfoDialog = document.createElement('template');
+  templateInfoDialog.innerHTML = `
+  <dialog id="${DIALOG_ID}">
+    <div class="header">
+      <h2>Keyboard Shortcuts</h2>
```

Comment:
```suggestion
      <h2>About SkipTo.js</h2>
```

It seems keyboard shortcuts have been disabled in this configuration, so the hardcoded “Keyboard Shortcuts” heading here doesn’t seem to apply? How about a more general heading matching the label of the trigger button?

Reply:
I will get that fixed, thanks for finding it!